src/System.Text.RegularExpressions/src/System/Text/RegularExpressions/GroupCollection.cs

```diff
@@ -250,5 +250,69 @@ void IEnumerator.Reset()
 
             void IDisposable.Dispose() { }
         }
+
+        public bool TryGetValue(string key, out Group value)
```

Comment:
Nit: all of the new methods should be moved up above the nested enumerator types.

Reply:
fixed